### PR TITLE
Add administration for header banner

### PR DESCRIPTION
**Pull request by illusionOfParadise**

### CODE REVIEW

File: src/Controller/BannerController.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+class BannerController extends Controller
+{
+    /**
+     * @Route("/header-banner")
+     * @Method("GET")
+     */
+    public function showHeaderBannerAction(): Response
+    {
+        return $this->render('banner/call.html.twig', [
+            'banner' => $this->getDoctrine()->getRepository('AppBundle:HomeBlock')->findOneBy(['position' => 11]),
```
Comment (HeahDude) [inline src/Controller/BannerController.php]: Please use FQCN `HomeBlock::class` instead of doctrine notation.

File: src/Entity/HomeBlock.php [hunk 2]
```diff
@@ -158,83 +158,69 @@ public function getPosition(): ?int
         return $this->position;
     }
 
-    public function setPosition(int $position): self
+    public function setPosition(int $position): void
     {
         $this->position = $position;
-
-        return $this;
```
Comment (HeahDude) [inline src/Entity/HomeBlock.php]: Are you sure this is not breaking anything?

Comment (illusionOfParadise) [inline src/Entity/HomeBlock.php]: Apparently, yes. I checked that there is no another method call of entity after.
Other suggestions why can it break anything ?

Comment (hhamon) [inline src/Entity/HomeBlock.php]: Method calls chaining in Doctrine just sucks! There are no reasons for returning `$this` in setter methods just to provide a fluent interface. So better to remove them as done in this PR and update locations where methods are chained.

**Pull request merged**